src/ComponentFolders/LoginPage/Login.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import Image from './Images/LoginSofa.svg';
+import lineImage from './Images/line.svg';
```

Comment:
![Screenshot_20221228_191951](https://user-images.githubusercontent.com/110013797/209843005-035902d2-76ab-4bc4-ad96-b447ae2a8cf0.png)
The line image doesn't seem to be there, maybe you deleted it by mistake before committing, can you add it again?